### PR TITLE
Merge new pipeline updates

### PR DESCRIPTION
**Pull request by howardxu520**

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -2,7 +2,8 @@ DB_N: "/projects/ps-yeolab3/bay001/annotations/nr/nt"
 DB_X: "/projects/ps-yeolab3/bay001/annotations/nr/nr.dmnd"
 
 
-N_downsample_reads: 10000
+N_downsample_reads: 100000
 
 SAMPLESPATH:
-  - "/oasis/tscc/scratch/s5xu/share/LARP6.CTRL_IN1.umi.r1.fqdown.genome-mapped.bam"
+  - "/home/bay001/projects/adri_chimeric_clips_20220811/temporary_data/crispr_clip_cas711_se_GRCh38_v2_small/results/Cas711_SE_GRCh38.EEF1A1_22nt_Cas711_IP_r1.umi.r1.fq.genome-mapped.bam"
```
Comment (byee4) [inline config.yaml:8]: If possible, let's keep the example config as-is, and point to small data/files that we've already created

File: scripts/blastresults_piechart_old.py
Comment (byee4) [inline scripts/blastresults_piechart_old.py:1]: Don't need to make copies of "old" since they'll be stored in your testing branch

File: examples.txt
Comment (byee4) [inline examples.txt:1]: Remove

File: jupyter_testing/IP_n_interest.csv
Comment (byee4) [inline jupyter_testing/IP_n_interest.csv:1]: Remove

File: jupyter_testing/another.ipynb
Comment (byee4) [inline jupyter_testing/another.ipynb:1]: This should stay in your testing branch

File: jupyter_testing/error_log.txt
Comment (byee4) [inline jupyter_testing/error_log.txt:1]: maybe have a "Jupyter testing" branch where all of your Jupyter notebook stuff can live, and keep your dev_improve branch to only improvement/additions to the actual scripts

**Review by byee4 — CHANGES_REQUESTED** [review]
Hi Shuhao, I reviewed your PR and I think your changes are good, but would like to clean up/migrate your Jupyter notebook stuff over to somewhere else. See if you can limit the merge to only improvements to the scripts that will be used by your Snakefile

**Comment by howardxu520** [comment]
OK, Let me keep all my debugging materials into a folder and put that folder under gitignore.

**Pull request closed without merging**